README.md

```diff
@@ -30,6 +36,10 @@ as well as the final prettier-ruby result by appending `DEBUG=true` when running
 DEBUG=true yarn run test --watch
 ```
 
+You should now see some useful debug output in your console:
+
+![Debug mode enabled, showing the parsed Ruby tokens and more](./documentation/debug-example.png)
```

Comment:
This might be a bit too big/distracting for the main read me. I'm happy to take it out if that's the case! :)